modules/currency-l1/src/test/scala/org/tessellation/currency/l1/MockDataApplicationL1Service.scala

```diff
@@ -0,0 +1,75 @@
+package org.tessellation.currency.l1
+
+import cats.data.NonEmptyList
+import cats.effect.IO
+
+import org.tessellation.currency.dataApplication.DataState.Base
+import org.tessellation.currency.dataApplication._
+import org.tessellation.currency.dataApplication.dataApplication.{DataApplicationBlock, DataApplicationValidationErrorOr}
+import org.tessellation.routes.internal.ExternalUrlPrefix
+import org.tessellation.schema.SnapshotOrdinal
+import org.tessellation.security.hash.Hash
+import org.tessellation.security.signature.Signed
+
+import io.circe.{Decoder, Encoder}
+import org.http4s.circe.CirceEntityCodec.circeEntityDecoder
+import org.http4s.circe.jsonEncoderOf
+import org.http4s.{EntityDecoder, EntityEncoder, HttpRoutes}
+
+object MockDataApplicationL1Service {
```

Comment:
It's actually a stub :)

Reply:
Maybe you can name it Dummy...Service, DummyUpdate, and so on 🤔

Reply:
renamed to dummy prefix